src/refda/amm/const.h

```diff
@@ -45,6 +45,9 @@ typedef struct
      */
     cace_ari_t value;
 
+    /** Determine whether the const has been marked obsolete
+     */
+    bool obsolete;
```

Comment:
Could this marking be moved up to the base object descriptor? Not a critial change needed here, but I think per the AMM that any objects can be marked as obsolete (which would also flow to a CAMP update). Can address this in a separate ticket if you want.

Reply:
That's a great idea, and I would prefer to address separately. Let me add a ticket.